src/Apps/Order/Utils/trackPageViewWrapper.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+import { trackPageView } from "./trackPageView"
+
+export function trackPageViewWrapper<T>(
```

Comment:
HOC's like this can be used as top-level decorators; since we're already using decorators best to keep things consistent.